nature/models.py

```diff
@@ -598,26 +598,26 @@ def __str__(self):
         return str(self.criterion)
 
 
-class EventRegulation(models.Model):
-    """Through model for Event & Regulation m2m relation"""
-    event = models.ForeignKey('Event', models.CASCADE, db_column='tapid')
+class TransactionRegulation(models.Model):
+    """Through model for Transaction & Regulation m2m relation"""
+    transaction = models.ForeignKey('Transaction', models.CASCADE, db_column='tapid')
     regulation = models.ForeignKey(Regulation, models.CASCADE, db_column='saaid')
 
     class Meta:
         db_table = 'tap_saados'
-        unique_together = (('event', 'regulation'),)
+        unique_together = (('transaction', 'regulation'),)
 
 
-class Event(ProtectionLevelMixin, models.Model):
+class Transaction(ProtectionLevelMixin, models.Model):
     register_id = models.CharField(max_length=20, blank=True, null=True, db_column='diaarinro')
     description = models.CharField(max_length=255, blank=True, null=True, db_column='kuvaus')
-    event_type = models.ForeignKey('EventType', models.PROTECT, db_column='tapahtumatyyppiid')
+    transaction_type = models.ForeignKey('TransactionType', models.PROTECT, db_column='tapahtumatyyppiid')
     last_modified_by = models.CharField(max_length=20, blank=True, null=True, db_column='paivittaja')
     date = models.DateField(blank=True, null=True, db_column='pvm')
     person = models.ForeignKey(Person, models.PROTECT, db_column='hloid', blank=True, null=True)
     link = models.CharField(max_length=4000, blank=True, null=True, db_column='linkki')
-    features = models.ManyToManyField(Feature, through='EventFeature', related_name='events')
-    regulations = models.ManyToManyField(Regulation, through='EventRegulation', related_name='events')
+    features = models.ManyToManyField(Feature, through='TransactionFeature', related_name='transaction')
+    regulations = models.ManyToManyField(Regulation, through='TransactionRegulation', related_name='transaction')
```

Comment:
The same as above

Reply:
oh, that was wired, sorry for that, I did a search and replace so I did not think this would happen, sorry, I will check a bit more before I do the PRs 😄